charts/posthog/values.yaml

```diff
@@ -586,6 +586,25 @@ clickhouse:
   # servers:
   # - host: posthog-posthog-zookeeper
   #   port: 2181
+  backup:
+    enabled: true
+    backup_user: backup
+    # PASSWORD=backup_password; echo "$PASSWORD"; echo -n "$PASSWORD" | sha256sum | tr -d '-'
+    backup_password: eb94c11d77f46a0290ba8c4fca1a7fd315b72e1e6c83146e42117c568cc3ea4d
```

Comment:
As we usually discourage to use passwords in the `values.yaml`, do you think it could be possible to pass here a reference (e.g. the name) of a k8s secret?

Reply:
sure, this is possible, I'll update my pr

Reply:
Yes but it could be good to expose both options:

- password provided via `values.yaml` (this creates a K8S secret that can be referenced around)
- password provisioned "externally" and reference provided via `values.yaml` (this doesn't create a K8S secret, it only uses the reference)

For few examples, take a look at our PostgreSQL or Redis config

Reply:
@guidoiaquinti take al look please, I want to have separate clickhouse user for taking backup, base on this article
https://github.com/AlexAkulov/clickhouse-backup/blob/master/Examples.md#how-to-use-clickhouse-backup-in-kubernetes
and at this moment current chart using the same approach for creating clickhouse user
https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/values.yaml#L579
https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/templates/clickhouse_instance.yaml#L9